### PR TITLE
Applicant Profile || front end page

### PR DESCRIPTION
**Pull request by ahmadysalah**
- [ ]  get all data for user in cookies from the database 

- [ ] render all data in table 

-to-do

### CODE REVIEW

**Comment by Asmaathabet** [comment]
don't make profile page as component , we will not use it anywhere as it is by this way, maybe we will have other additions to this page, editing email and password for example and other things.

File: client/src/components/pages/application/Profile/index.js [hunk 1]
```diff
@@ -1,5 +1,329 @@
...
+                      Submitted
+                    </Typography>
+                  </td>
+                  <td />
+                </tr>
+                <tr>
+                  <td className="item">
+                    <Typography variant="body2" align="left">
+                      Code Wares Score
+                    </Typography>
```
Comment (Asmaathabet) [inline client/src/components/pages/application/Profile/index.js:251]: Typo 
```suggestion
                    <Typography variant="body2" align="left">
                      codewars Score
                    </Typography>
```

File: client/src/components/pages/application/Profile/index.js [hunk 2]
```diff
@@ -1,5 +1,329 @@
...
+                    <Typography variant="body2" color="primary" align="left">
+                      {data.codeWarsKyu}
+                    </Typography>
+                  </td>
+                </tr>
+                <tr>
+                  <td className="item">
+                    <Typography variant="body2" align="left">
+                      FreeCode Camp Score
+                    </Typography>
```
Comment (Asmaathabet) [inline client/src/components/pages/application/Profile/index.js:263]:
```suggestion
                    <Typography variant="body2" align="left">
                      freecodecamp Score
                    </Typography>
```

File: client/src/components/pages/application/Profile/index.js [hunk 3]
```diff
@@ -1,5 +1,329 @@
...
+                  </td>
+                  <td>
+                    <Typography variant="body2" color="primary" align="left">
+                      {data.freeCodeCampPoints}
```
Comment (Asmaathabet) [inline client/src/components/pages/application/Profile/index.js:267]: numbers appears a bit lower than its name, work on its style, the same for codewars

File: client/src/components/pages/application/Profile/index.js [hunk 4]
```diff
@@ -1,5 +1,329 @@
...
+        alt="backGround"
+        className="profileImage profile"
+      />
+
+      {data && Technical ? (
+        <div className="Profile__container">
+          <Card
+            ClassName="card_profile"
+            content={
+              <table className="Profile__table">
```
Comment (alidah0) [inline client/src/components/pages/application/Profile/index.js:73]:
```suggestion
              <table className="Profile__table">
              <tbody> content </tbody>
              </table>
```
There is a 20 validateDOMNesting Error in the console 
just wrap the content inside the TABLE tag with TBODY tag

File: client/src/components/pages/application/Profile/index.js [hunk 5]
```diff
@@ -1,5 +1,329 @@
...
+                <tr className="doted">
+                  <td>
+                    <Typography variant="body2" color="primary" align="left">
+                      Technical Task
+                    </Typography>
+                  </td>
+                  <td />
+                </tr>
+                {Technical &&
+                  Technical.map(({ taskName }) => (
```
Comment (alidah0) [inline client/src/components/pages/application/Profile/index.js:192]:
```suggestion
                  Technical.map(({ taskName, _id }) => (
                      <tr key={_id}>
```
fix this line
I know you, It's a simple issue and you probably knew about this.

**Pull request merged**